### PR TITLE
Fixes UserFrosting Issue #990 - Can't set Cookie Domain of RememberMe Cookie

### PR DESCRIPTION
**Pull request by xrobau**
This allows the domain to be configured in the uf4-rememberme
cookie.

### CODE REVIEW

File: app/sprinkles/account/src/Authenticate/Authenticator.php [hunk 1]
```diff
@@ -122,6 +122,11 @@ public function __construct(ClassMapper $classMapper, Session $session, Config $
             $this->rememberMe->getCookie()->setExpireTime($this->config['remember_me.expire_time']);
         }
 
+        // Set domain, if specified
+        if ($this->config->has('remember_me.domain') && ($this->config->has('remember_me.domain') != null)) {
```
Comment (lcharette) [inline app/sprinkles/account/src/Authenticate/Authenticator.php]: Simple note, you have extra `()` pair in there : 

```
if ($this->config->has('remember_me.domain') && $this->config->has('remember_me.domain') != null) {
```

Comment (xrobau) [reply app/sprinkles/account/src/Authenticate/Authenticator.php]: I copied the coding style of the line above it.

**Comment by xrobau** [comment]
Done

**Comment by lcharette** [comment]
Oops, StyleCI doesn't agree with you :P 

https://github.styleci.io/analyses/qrnyOx?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

**Pull request merged**